pkgs/applications/misc/qtpass/default.nix

```diff
@@ -34,6 +34,8 @@ mkDerivation rec {
     })
   ];
 
+  sandboxProfile = qtbase.__sandboxProfile;
```

Comment:
In `qtkeychain` you added a comment explaining why we can't use `propagatedSandboxProfile`, might be in order here too?

Maybe we should open an issue on the sandbox profile not being propagated?

Reply:
I’ll add a comment to qtpass as well.